mozilla/fx-private-relay
# Add multi-reply tip

# New feature description

This adds instructions on how to reply to multiple senders.

# Screenshot (if applicable)

[See preview site.](https://deploy-preview-2781--fx-relay-demo.netlify.app/accounts/profile/?mockId=some)

# How to test

Open the email masking dashboard (adding it to the phone masking dashboard is in https://github.com/mozilla/fx-private-relay/pull/2780) with an account that has phone masking setup. You should see the multi-reply tip. You should not see it with a user that does not have a Relay number yet.

# Checklist

- [x] l10n changes have been submitted to the l10n repository, if any. https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/114
